modules/gitops-disable-dex.adoc

```diff
@@ -0,0 +1,17 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="disable-dex_{context}"]
+= Disabling Dex
+
```

Comment:
To adhere to Mod. Doc. procedure guidelines, consider the following restructure:

=Disabling Dex

Dex is installed by default for all the Argo CD instances created by the Operator. You can disable Dex. 

Procedure 

- Set the environmental variable `DISABLE_DEX` to true in the Subscription resource of the Operator:
<source, yaml you've provided>